crates/bevy_render/src/mesh/primitives/dim3/mod.rs

```diff
@@ -4,6 +4,7 @@ mod cylinder;
 mod plane;
 mod sphere;
 mod torus;
+pub(crate) mod triangle3d;
```

Comment:
why is this in a different format to the other module declarations?

Reply:
It's pub(crate) so that the `Triangle2d` code can see the `Triangle3d` UV generation code